app/lib/methods/handleImageBase64.ts

```diff
@@ -0,0 +1,5 @@
+export const imageBase64RegExp = new RegExp(/^data:image\/[bmp,gif,ico,jpg,png,svg,webp,x\-icon,svg+xml]+;base64,/);
+
+export function isImageBase64(data: string): boolean {
+	return !!data && imageBase64RegExp.test(data);
```

Comment:
You're allowing `string`, but you need `!!data` because of empty strings?

Wouldn't this work?
```suggestion
export function isImageBase64(data?: string): boolean {
	return imageBase64RegExp.test(data);
```